cmux-tui/spec/commands.md

```
@@ -289,15 +294,55 @@ CLI mapping:
+| `exclusive` | `boolean` | default `false` | Valid only with `client` and `enabled:true` |
+
+Result: `object{}`.
+
+Errors include `unknown surface <id>`, `client <id> is not attached to surface <id>`, `client <id> has no reported size for surface <id>`, and invalid exclusive or disabled-all combinations.
+
+CLI mapping:
+
+| Item | Value |
+| --- | --- |
+| Verb | `set-client-sizing` |
+| Flags | `--surface <id> --client <id> --enabled <true|false>` |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)